client_impl.go

```diff
@@ -138,16 +131,16 @@ func (c *clientImpl) OnlineQueryBulk(params OnlineQueryParamsComplete) (OnlineQu
 	)
 
 	if err != nil {
-		return emptyResult, getErrorResponse(err)
+		return emptyResult, errors.Wrap(err, "send request")
```

Comment:
```suggestion
		return emptyResult, errors.Wrap(err, "sending request")
```

nit, more consistent with rest of the tenses